doc/swss-schema.md

```diff
@@ -630,6 +634,35 @@ Equivalent RedisDB entry:
 
 ## Configuration DB schema
 
+### PORT_TABLE
+Stores information for physical switch ports managed by the switch chip. Ports to the CPU (ie: management port) and logical ports (loopback) are not declared in the PORT_TABLE. See MGMT_PORT.
+
+    ;Configuration for layer 2 ports
+    port_table_key      = PORT|ifname    ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    admin_status        = BIT           ; is the port enabled (1) or disabled (0)
+    lanes               = list of lanes ; (need format spec???)
+    ifname              = 1*64VCHAR     ; name of the port, must be unique
```

Comment:
why do we need this? what's the difference between this and the key of the entry?

Reply:
removed